vm_trace.c

```diff
@@ -1517,6 +1517,25 @@ tracepoint_stat_s(rb_execution_context_t *ec, VALUE self)
     return stat;
 }
 
+static VALUE
+restore_trace_arg(VALUE val)
+{
+    rb_trace_arg_t *arg = (rb_trace_arg_t *)val;
+    rb_execution_context_t *ec = GET_EC();
+    if (ec->trace_arg != NULL) rb_bug("should be NULL, but %p", ec->trace_arg);
```

Comment:
Simple curiosity, why use an explicit `if () rb_bug` instead of something like `RUBY_ASSERT`?

Reply:
I want to show the `ec->trace_arg` (and `assert` doesn't show).

Reply:
Thank you!